doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -201,12 +201,24 @@ Automated metadata generation
 By enabling the ``SB_CONFIG_DFU_ZIP_BLUETOOTH_MESH_METADATA`` option in sysbuild, the metadata will be automatically parsed from the ``.elf`` and ``.config`` files.
 The parsed data is stored in the :file:`ble_mesh_metadata.json` file.
 The file is placed in the :file:`dfu_application.zip` archive in the build folder of the application.
-Additionally, the metadata string required by the ``mesh models dfu slot add`` command will be printed in the command line window when the application is built::
+Additionally, the metadata string and firmware ID required by the ``mesh models dfu slot add`` command will be printed in the command line window when the application is built::
 
   Bluetooth Mesh Composition metadata generated:
     Encoded metadata: 020000000000000094cf24017c26f3710100
+    Firmware ID: 59000200000000000000
     Full metadata written to: APPLICATION_FOLDER\build\zephyr\dfu_application.zip
 
+The Firmware ID can be generated from one of multiple sources:
+
+  * To use a user-supplied hex-string as the Firmware ID, enable the ``SB_CONFIG_DFU_ZIP_BLUETOOTH_MESH_METADATA_FWID_CUSTOM`` option. This is the default behavior.
```

Comment:
@peknis For some reason this line ends up in boldface. I don't think I understand why, do you have any idea? (See doc build output, https://github.com/nrfconnect/sdk-nrf/pull/20255)

Reply:
Yep, my bad. Start the first bullet from the very beginning of the line and the sub bullet as usual from the beginning of the previous bullet's text.